src/alarm_trap_sender.cpp

```diff
@@ -47,6 +47,8 @@
 #include "itu_alarm_table.hpp"
 #include "alarm_active_table.hpp"
 
+std::string MIB_version = "201608081100";
```

Comment:
Please comment what this is (and when it needs changing), capitals as this is a constant